ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/FastSyncDownloader.java

```diff
@@ -142,7 +143,7 @@ public void stop() {
     synchronized (this) {
       if (running.compareAndSet(true, false)) {
         LOG.info("Stopping sync");
-        // Cancelling the world state download will also cause the chain download to be cancelled.
+        // Canceling the world state download will also cause the chain download to be cancelled.
```

Comment:
I had to look it up. US tends to favor the single "l" version ("canceling" and "canceled") but both are used. However, the double ll version is used in a few spots so if you want to do a find and replace I won't object but it could be a separate PR